cirq-google/cirq_google/serialization/common_serializers_test.py

```diff
@@ -185,24 +183,11 @@ def test_half_pi_does_not_serialize_arbitrary_xy():
 
 @pytest.mark.parametrize(
     ('x_exponent', 'z_exponent', 'axis_phase_exponent'),
-    [
-        (0, 0, 0),
-        (1, 0, 0),
-        (0, 1, 0),
-        (0.5, 0, 0.5),
-        (0.5, 0.5, 0.5),
-        (0.25, 0.375, 0.125),
-    ],
+    [(0, 0, 0), (1, 0, 0), (0, 1, 0), (0.5, 0, 0.5), (0.5, 0.5, 0.5), (0.25, 0.375, 0.125)],
```

Comment:
I think this is a little clearer line-by-line as well.

Reply:
Fixed by adding one more test case.